gradle-plugins/android-sdk/downloadAndroidSdk

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+
+clone() {
+    mkdir -p $1
+    pushd $1
+    git init
+    git config advice.detachedHead false
+    git fetch --depth=1 $2 $3
+    git checkout FETCH_HEAD
+    popd
+}
+
+# Commit hashes and sdk versions from https://android.googlesource.com/platform/manifest/+/refs/heads/androidx-main/default.xml
+
+downloadLinuxSDK() {
+    clone linux/platforms/android-32 https://android.googlesource.com/platform/prebuilts/fullsdk/platforms/android-32 master
```

Comment:
I just took version from core repository, but I get this kind of errors:
```
Could not determine the dependencies of task ':compileAndroidMain'.
> Failed to install the following Android SDK packages as some licences have not been accepted.
     build-tools;34.0.0 Android SDK Build-Tools 34
  To build this project, accept the SDK license agreements and install the missing components using the Android Studio SDK Manager.
  All licenses can be accepted using the sdkmanager command line tool:
  sdkmanager.bat --licenses
```

Reply:
Seems like we need to change SDK to 34, but I don't see repositories with that.
Or we need to change SDK to 32 in test projects. But it didn't help me
@igordmn do you have any thoughts?